app/models/automation_request.rb

```diff
@@ -37,6 +37,13 @@ def self.create_from_ws(version, user, uri_parts, parameters, requester)
     create_request(options, user, auto_approve)
   end
 
+  def self.create_from_scheduled_task(user, uri_parts, parameters)
+    [:namespace, :class_name].each { |key| uri_parts.delete(key) if uri_parts.key?(key) }
+    approval = {'auto_approve' => true}
+    uri_parts.stringify_keys!.reverse_merge!('namespace' => "/#{DEFAULT_NAMESPACE.capitalize}", 'class' => DEFAULT_CLASS.capitalize)
```

Comment:
@syncrou You do not need to call `capitalize` on namespace or class as the automate resolution is not case sensitive.  However, this line should not be needed since `namespace` and `class` will be defaulted in `create_from_ws `.